OracleCoherence/dockerfiles/14.1.1.0.0/src/main/docker/start.sh

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env sh
+
+#!/bin/sh -e -x -u
```

Comment:
You can't have two shebang lines. This line should be changed to `set -eux` instead